test/distributed/_composable/fully_shard/test_fully_shard_init.py

```diff
@@ -267,7 +267,7 @@ def test_nested_fully_shard_shared_state(self):
         # NOTE: This check only requires that the data structure state is
         # shared. Namely, sharing the FSDP state object itself is sufficient
         # but not necessary.
-        data_structure_names = ["_streams", "_exec_order_data", "_free_event_queue"]
+        data_structure_names = ["_exec_order_data", "_free_event_queue"]
```

Comment:
I realized we should follow-up to re-add the flattened stream data structures into this check.